apm-application-toolkit/apm-toolkit-trace/src/main/java/org/apache/skywalking/apm/toolkit/trace/ActiveSpan.java

```diff
@@ -26,10 +26,21 @@
  */
 public class ActiveSpan {
     /**
-     * @param key tag key
+     * @param key   tag key
      * @param value tag value
      */
     public static void tag(String key, String value) {
 
     }
+
+    public static void error() {
+
+    }
+
+    public static void error(String key, String errorMsg) {
```

Comment:
The core agent API accepts log as Map if we only log errorMsg, I prefer to just support one parameter, without `key`. Log
1. `add(new KeyValuePair("event", "error")`
1. `add(new KeyValuePair("message", xxxx)`.

Take a look at https://github.com/apache/skywalking/blob/3eafda2865da0be3d701515bd261ce2934fdf802/apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/trace/AbstractTracingSpan.java#L149-L152

Reply:
got it